### ISSUE TITLE
how to fine tune and resume the model？

### ISSUE THREAD

**Issue by sinason** [yijingru/BBAVectors-Oriented-Object-Detection#20]

**Comment by yijingru** [comment]
I guess may split a little dataset as a validation set to train the model and use the epoch that achieves the best accuracy (or lowest loss). Then train the whole training set (include the previous validation set) to train again and test at that epoch. I don't understand the second question, resume model in the training?

**Comment by sinason** [comment]
The second question is if the training is interrupted for some reason, how can I restart the training using the most recent checkpoint model?
As for the loss, due to the GPU, my epoch is small, and loss will not change in 2.x any more. The accuracy cannot be measured on the test set without labels, so I would like to ask about the final approximate loss range of your model in the paper.

**Comment by sinason** [comment]
> I guess you may split a little dataset as a validation set to train the model and use the epoch that achieves the best accuracy (or lowest loss, or when the loss is noticeably decreasing). Then train the whole training set (include the previous validation set) again and test at that epoch. I don't understand the second question, resume model in the training?

The second question is if the training is interrupted for some reason, how can I restart the training using the most recent checkpoint model?
As for the loss, due to the GPU, my epoch is small, and loss will not change in 2.x any more. The accuracy cannot be measured on the test set without labels, so I would like to ask about the final approximate loss range of your model in the paper.

**Comment by yijingru** [comment]
I see. I would like to add the resume part in `train.py`. One suggestion is to uncomment (use) `nn.BatchNorm2d(head_conv)` in `ctrbox_net.py` to help stable training.

As for loss, I checked my logs, for either 20 or 48 batch sizes, the loss ranges from 1 to 4. The larger batch size has a smaller loss.  

![image](https://user-images.githubusercontent.com/25541074/96300005-e1206200-0fc2-11eb-883f-23ab4712a405.png)

**Comment by sinason** [comment]
> I see. I would like to add the resume part in `train.py`. One suggestion is to uncomment (use) `nn.BatchNorm2d(head_conv)` in `ctrbox_net.py` to help stable training.
> 
> As for loss, I checked my logs, for either 20 or 48 batch sizes, the loss ranges from 1 to 4. The larger batch size has a smaller loss.
> 
> ![image](https://user-images.githubusercontent.com/25541074/96300005-e1206200-0fc2-11eb-883f-23ab4712a405.png)

Thanks a lot!
